sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -195,6 +217,21 @@ def from_connection_string(cls, conn_str, consumer_group, **kwargs):
         :paramtype checkpoint_store: ~azure.eventhub.CheckpointStore
         :keyword float load_balancing_interval: When load-balancing kicks in. This is the interval, in seconds,
          between two load-balancing evaluations. Default is 10 seconds.
+        :keyword float partition_ownership_expiration_interval: A partition ownership will expire after this number
+         of seconds. Every load-balancing evaluation will automatically extend the ownership expiration time.
+         Default is 6 * load_balancing_interval, i.e. 60 seconds when using the default load_balancing_interval
+         of 10 seconds.
+        :keyword str or ~azure.eventhub.LoadBalancingStrategy load_balancing_strategy: When load-balancing kicks in,
+         it will use this strategy to claim and balance the partition ownership.
+         Use "greedy" or `LoadBalancingStrategy.GREEDY` for the greedy strategy, which, for every
+         load-balancing evaluation, will grab all partitions that are not claimed by any other `EventHubConsumerClient`.
```

Comment:
do we need to update the docstring? -- reasonable greedy, not all partitions

Reply:
Good catch. Updated.